Translation/en/RedirectedResources/assets/abdata/communication/info_00/topiclistenb_14/translation.txt

```diff
@@ -2,38 +2,38 @@
 // Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.1
 //
 //「<color%3Dyellow>ツリーイング</color>って面白いよな」=
```

Comment:
We'll need to come back around to these still, shouldn't be too bad.

Reply:
Once everything is merged for both KK and KKS I'll do final passes to fill in any missing translations.